### DISCUSSION TITLE
Change canvas level and camera size manually

### DISCUSSION THREAD

**Discussion by F-Burning** [opentoonz/opentoonz#6448] [Ideas]
This is in Clip studio paint.

https://user-images.githubusercontent.com/74098734/158825680-a979ab13-be45-417b-962e-8b7023f39a0e.mp4

**Comment by artisteacher** [comment]
Yes!

**Comment by RodneyBaker** [comment]
Converting this to Discussions so others can attempt to see a way forward to implementing this.
A slightly similar capability can be achieved via the Camera Settings 'Use Current Level' option to resize the Camera and (once merged) @Abel032's Layout feature enhances that capability.

In short, if we can load a specifically sized image we can conform the camera to that image.

Again, not as elegant and direct as just resizing via the corners but perhaps an interim workaround while awaiting a full solution.
